File: pset6/mario.py
import cs50

while True:
    print("Height: ", end = "")
    height = cs50.get_int()
    if height > 0 and height < 23:
        break
    
for row in range(height):
    print(" " * (height - row - 1), end = "")
    print("#" * (row + 1), end = "")
    print("  ", end = "")
    print("#" * (row + 1))

File: pset1/mario/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void){
    
    // initialize height variable
    int height;
    
    // prompt user for height and repeat until valid number given
    do{
        printf("Height: ");
        height = get_int();
    }
    while(height > 23 || height < 0);
    
    // construct pyramid
    for(int row = 0; row < height; row++){

        // add padding spaces to row
        for(int i = 0; i < (height - row - 1); i++){
            printf(" ");
        }
        
        // add first pyramid's hashes to row
        for(int i = 0; i <= row; i++){
            printf("#");
        }
        
        // add middle spaces to row
        printf("  ");
        
        // add second pyramid's hashes to row
        for(int i = 0; i <= row; i++){
            printf("#");
        }
        
        // terminate line with newline character
        printf("\n");
    }
}